restler/engine/core/request_utilities.py

```diff
@@ -20,6 +23,7 @@
 from engine.transport_layer.response import RESTLER_BUG_CODES
 from engine.transport_layer.messaging import UTF8
 from engine.transport_layer.messaging import HttpSock
+from engine.core.retry_handler import RetryHandler
 
 last_refresh = 0
 NO_TOKEN_SPECIFIED = 'NO-TOKEN-SPECIFIED\r\n'
```

Comment:
Require a constants file, if this potentially used elsewhere?

Reply:
Changed retry handler to use enums for different strategies